Documentation/deploy-workers.md

```diff
@@ -194,24 +258,30 @@ Tell systemd to rescan the units on disk:
 $ sudo systemctl daemon-reload
 ```
 
-### Start kubelet
+### Start kubelet, flannel and Calico Node
 
-Start the kubelet, which will start the proxy as well.
+Start the kubelet, which will start the proxy as well and start the Calico node.
 
 ```sh
+$ sudo systemctl start flanneld
 $ sudo systemctl start kubelet
+$ sudo systemctl start calico-node
 ```
 
-Ensure that the kubelet starts on each boot:
+Ensure that the services start on each boot:
 
 ```sh
+$ sudo systemctl enable flanneld
+Created symlink from /etc/systemd/system/multi-user.target.wants/flanneld.service to /etc/systemd/system/flanneld.service.
 $ sudo systemctl enable kubelet
 Created symlink from /etc/systemd/system/multi-user.target.wants/kubelet.service to /etc/systemd/system/kubelet.service.
+$ sudo systemctl enable calico-agent
+Created symlink from /etc/systemd/system/multi-user.target.wants/calico-agent.service to /etc/systemd/system/calico-agent.service.
+
 ```
 
 To check the health of the kubelet systemd unit that we created, run `systemctl status kubelet.service`.
-
-If you run into issues with Docker and Flannel, check to see that the drop-in was applied correctly by running `systemctl cat docker.service` and ensuring that the drop-in appears at the bottom.
+To check the health of the calico-node systemd unit that we created, run `systemctl status calico-node.service`.
```

Comment:
This section intermixes use of `calico-node` and `calico-agent`. Not sure which is valid, but should unify naming.